Dataset.py

```diff
@@ -545,6 +570,8 @@ def init_seq_order(self, epoch=None, seq_list=None):
 
     This is called when we start a new epoch, or at initialization.
     Call this when you reset the seq list.
+
+    TODO: it does not seem to be obvious what the sub-classes do/should do when this function is caleld
```

Comment:
Well, then extend the documentation if you think it's unclear?

Reply:
This is still unclear for me, so I am not sure if I can do it.

Reply:
I don't exactly understand. What is unclear?

Reply:
This is called when we start a new epoch, or at initialization. The subclasses can do whatever they want here.
Usually they would initialize the seq order, maybe do some other preparation for the epoch (whatever they want).